lib/transforms/json-to-html-data.js

```diff
@@ -0,0 +1,143 @@
+module.exports = (resource) => {
+  const title = getTitle(resource.data);
+  const type = getType(resource.data);
+  const fact = getFacts(resource.data);
+  const related = getRelated(resource);
+  const description = getDescription(resource.data);
+  const details = getDetails(resource.data);
+
+  return {title, type, fact, related, description, details};
+};
+
+function getTitle (data) {
+  return data.attributes.summary_title;
+}
+
+function getType (data) {
+  return data.type;
+}
+
+function getFacts (data) {
+  // Need confirmation of what attributes to use as facts
+  const occupation = getOccupation(data);
+  const nationality = getNationality(data);
+  const born = getBorn(data);
+  const birth = getBirth(data);
+  const death = getDeath(data);
+
+  return [occupation, nationality, born, birth, death].filter(el => el);
+}
+
+function getOccupation (data) {
+  try {
+    if (data.attributes.occupation) {
+      return {key: 'Occupation', value: data.attributes.occupation};
+    } else {
+      throw new Error('No Occupation');
+    }
+  } catch (err) {
+    return null;
+  }
```

Comment:
What's the reason for using try/catch like this? We should be using try/catch for exceptions, meaning simply that I don't think we should regard properties that legitimately may or may not exist on an object as an exception.